services/repository-managers/src/test/java/org/sagebionetworks/repo/manager/dataaccess/AccessRequirementManagerImplUnitTest.java

```diff
@@ -480,6 +510,7 @@ public void testUpdateConcreteType() {
 		Assertions.assertThrows(IllegalArgumentException.class, () -> {
 			arm.updateAccessRequirement(userInfo, accessRequirementId, toUpdate);
 		});
+		verify(mockTransactionalMessenger).sendMessageAfterCommit(TEST_ENTITY_ID, ObjectType.ENTITY, ChangeType.UPDATE);
```

Comment:
missing case where there are multiple subjects and case where subject is a folder or project.  Will need a test for the delta on update.